docs/versioning.rst

```diff
@@ -0,0 +1,89 @@
+**********
+Versioning
+**********
+
+
+Current Status
+==============
+
+.. list-table:: Releases
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Release
+     - Supported Runtimes
+     - Minimum Trace Agent
+     - Status
+   * - 0.x
+     - CPython: 2.7, 3.5+
+     - 
+     - MAINTENANCE [2022-02-25]
+   * - 1.x
+     - CPython: 2.7, 3.5+
```

Comment:
```suggestion
     - CPython: 3.6+
```
🙏

Reply:
Not yet... but I think we can do this with a planned 2.0 release.

My thinking is we can introduce these terms, maintenance definitions for 1.0, so that way when we do drop support for 2.7 people understand what they are going to get/how we are going to continue to support them going forward.